include/nlohmann/detail/meta/type_traits.hpp

```diff
@@ -94,6 +94,12 @@ using get_template_function = decltype(std::declval<T>().template get<U>());
 template<typename BasicJsonType, typename T, typename = void>
 struct has_from_json : std::false_type {};
 
+template <typename BasicJsonType, typename T>
```

Comment:
Could you please add the rationale why this function is needed rather than `std::is_constructible`?

Reply:
I've added the rationale [here](https://github.com/nlohmann/json/pull/1559#issuecomment-661111824)

Reply:
Do you want me to add it in the code?

Reply:
Yes, please. Such things are so easy forgotten...